client/packages/common/src/types/schema.ts

```diff
@@ -2964,6 +2965,11 @@ export type StoreNode = {
   __typename: 'StoreNode';
   code: Scalars['String'];
   id: Scalars['String'];
+  /**
```

Comment:
Not sure why this was generated now... didn't go through with the associated PR?